manage.py

```
@@ -4,6 +4,9 @@
 import sys
 import subprocess
 from pathlib import Path
+import sys
```

Ruff (0.14.5)

Redefinition of unused `sys` from line 4

Remove definition: `sys`

(F811)